src/config_file.cc

```diff
@@ -264,6 +280,9 @@ void CONFIG_TREC::setup_config( string filename ) {
   lmparam_smoothing1 = -1.0;
   lmparam_smoothing2 = -1.0;
   lmparam_mixture = -1.0;
+  normalizations = "ntn";
```

Comment:
Length check is already present in `check_tfidf()` so just setting `normalizations` to default empty string with 239f179.